account_payment_return_financial_risk/tests/test_payment_return_risk.py

```diff
@@ -30,14 +27,19 @@ def setUpClass(cls):
             {
                 "name": "Test income account",
                 "code": "INCOME",
-                "user_type_id": cls.env["account.account.type"]
-                .create(
-                    {"name": "Test income", "type": "other", "internal_group": "income"}
-                )
-                .id,
+                "account_type": "income_other",
             }
         )
         cls.partner = cls.env["res.partner"].create({"name": "Test"})
+        cls.tax = cls.env["account.tax"].create(
```

Comment:
Why do you need a tax?

Reply:
A 0% rate is used to prevent Tax 15% from self-defining and the amounts are not as expected.

Reply:
@victoralmau If you force tax_ids = False? still happens the same?

Reply:
Then just empty the field, instead of creating a new one.